### PR TITLE
add v0.1.13 release notes

### PR DESCRIPTION
**Pull request by hengm3467**
## Info
- **Description**: 
Added the release notes for v0.1.13. The release date needs to be updated once finalized.

See the preview here: https://release-notes-0-1-13.d2fbku9n2b6wde.amplifyapp.com/docs/latest/release-notes/

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>image</details>

## Published doc pages
  [ After merging this PR, edit the description to include the links to the updated doc pages here. For example, https://www.risingwave.dev/docs/latest/intro/. ]

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,66 @@ slug: /release-notes
...
+    - Casting from varchar to integer allows leading and trailing spaces. [#5452](https://github.com/risingwavelabs/risingwave/pull/5452)
+        
+        `' 1 '::int` → `1`
+        
+- New psql meta-commands are supported.
+    - `\d`: List all relations in the current database, together with indexes (originally not supported in psql). (Materialized) sources are not supported yet.
+    - `\dt`: List all tables in the current database.
+    - `\dm`: List all materialized views in the current database.
+    
+    ![Untitled](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/6b9ad16f-ff72-448e-856d-743e7093992b/Untitled.png)
```
Comment (fuyufjh) [inline docs/release-notes.md]:
```suggestion
    ![](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/6b9ad16f-ff72-448e-856d-743e7093992b/Untitled.png)
```

**Review by fuyufjh — APPROVED** [review]

File: docs/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,66 @@ slug: /release-notes
...
+        `array[1,2]::varchar` → `{1,2}`
+        
+    - Casting from varchar to integer allows leading and trailing spaces. [#5452](https://github.com/risingwavelabs/risingwave/pull/5452)
+        
+        `' 1 '::int` → `1`
+        
+- New psql meta-commands are supported.
+    - `\d`: List all relations in the current database, together with indexes (originally not supported in psql). (Materialized) sources are not supported yet.
+    - `\dt`: List all tables in the current database.
+    - `\dm`: List all materialized views in the current database.
```
Comment (yezizp2012) [inline docs/release-notes.md]:
```suggestion
    - `\dm`: List all materialized views in the current database.
    - `\di`: List all indexes in the current database.
```

File: docs/release-notes.md [hunk 3]
```diff
@@ -7,6 +7,66 @@ slug: /release-notes
...
+    - Casting from array to varchar is now supported. [#5081](https://github.com/risingwavelabs/risingwave/pull/5081)
+        
+        `array[1,2]::varchar` → `{1,2}`
+        
+    - Casting from varchar to integer allows leading and trailing spaces. [#5452](https://github.com/risingwavelabs/risingwave/pull/5452)
+        
+        `' 1 '::int` → `1`
+        
+- New psql meta-commands are supported.
+    - `\d`: List all relations in the current database, together with indexes (originally not supported in psql). (Materialized) sources are not supported yet.
```
Comment (yezizp2012) [inline docs/release-notes.md]:
```suggestion
    - `\d`: List all relations in the current database. (Materialized) sources are not supported yet.
```

File: docs/release-notes.md (hunk 2 above)
Comment (yezizp2012) [reply docs/release-notes.md]: A new command `\di` is introduced recently in https://github.com/risingwavelabs/risingwave/pull/5742 , together with a new system table `pg_catalog.pg_index`.

**Review by xiangjinwu — APPROVED** [review]

**Review by lmatz — APPROVED** [review]

**Review by yezizp2012 — APPROVED** [review]

**Review by skyzh — APPROVED** [review]

**Pull request merged**